src/python/tensorflow_cloud/core/README.md

```diff
@@ -0,0 +1,413 @@
+- [Using `run` within a python script that contains the `tf.keras` model](https://github.com/tensorflow/cloud/blob/master/src/python/tensorflow_cloud/core/tests/examples/call_run_within_script_with_keras_fit.py).
+- [Using cloud build instead of local docker](https://github.com/tensorflow/cloud/blob/master/src/python/tensorflow_cloud/core/tests/examples/call_run_on_script_with_keras_fit_cloud_build.py).
+- [Run AutoKeras with TensorFlow Cloud](https://github.com/tensorflow/cloud/blob/master/src/python/tensorflow_cloud/core/tests/examples/call_run_within_script_with_autokeras.py).
+
+## Running unit tests
+
+```shell
+pytest src/python/tensorflow_cloud/core/tests/unit/
+```
+
+## Local vs remote training
+
+Things to keep in mind when running your jobs remotely:
+
+[Coming soon]
+
+## Debugging workflow
+
+Here are some tips for fixing unexpected issues.
+
+### Operation disallowed within distribution strategy scope
+
+**Error like**: Creating a generator within a strategy scope is disallowed, because there is ambiguity on how to replicate a generator (e.g. should it be copied so that each replica gets the same random numbers, or 'split' so that each replica gets different random numbers).
+
+**Solution**: Passing `distribution_strategy='auto'` to `run` API wraps all of your script in a TF distribution strategy based on the cluster configuration provided. You will see the above error or something similar to it, if for some reason an operation is not allowed inside distribution strategy scope. To fix the error, please pass `None` to the `distribution_strategy` param and create a strategy instance as part of your training code as shown in [this](https://github.com/tensorflow/cloud/blob/master/src/python/tensorflow_cloud/core/tests/testdata/save_and_load.py) example.
+
+### Version not supported for TPU training
+
+**Error like**: There was an error submitting the job.Field: tpu_tf_version Error: The specified runtime version '2.3' is not supported for TPU training. Please specify a different runtime version.
+
+**Solution**: Please use TF version 2.1. See TPU Strategy in [Cluster and distribution strategy configuration section](#cluster-and-distribution-strategy-configuration).
+
+### TF nightly build.
+
+**Warning like**: Docker base image '2.4.0.dev20200720' does not exist. Using the latest TF nightly build.
+
+**Solution**: If you do not provide `docker_base_image` param, then by default we use pre-built TF docker images as base image. If you do not have TF installed on the environment where `run` is called, then TF docker image for the `latest` stable release will be used. Otherwise, the version of the docker image will match the locally installed TF version. However, pre-built TF docker images aren't available for TF nightlies except for the latest. So, if your local TF is an older nightly version, we upgrade to the latest nightly automatically and raise this warning.
+
+## Coming up
+
+- Distributed Keras tuner support.
+
+## Contributing
```

Comment:
because you have covered those in the top-level README.me, you can dispense those sections.

Reply:
Done.